.github/config/goreleaser.yaml

```
@@ -97,11 +97,12 @@ nfpms:
     vendor: "Open Component Model"
     homepage: "https://ocm.software/"
     maintainer: "OCM Project <open-component-model@noreply.github.com>"
-    description: "The OCM CLI makes it easy to create component versions and embed them in build processes."
+    description: |
+      "The OCM CLI is used to create and maintain component versions.
+      It can be used to easily embed OCM related actions in CICD processes."
```

If I am not mistaken, then you should not need the `"` on multiline-strings
```suggestion
    description: |
      The OCM CLI is used to create and maintain component versions.
      It can be used to easily embed OCM related actions in CICD processes.
```